### PR TITLE
BACK-7120: Add uniswapX EIP712 definitions

### PR DESCRIPTION
**Pull request by jnicoulaud-ledger**
from @lcastillo-ledger :)
supersedes #186

### CODE REVIEW

File: arbitrum/uniswap/eip712.json [hunk 1]
```diff
@@ -80,6 +80,451 @@
...
+                            {
+                                "assetPath": "witness.outputs[0].token",
+                                "format": "amount",
+                                "label": "Receive at Minimum",
+                                "path": "witness.outputs[0].amount"
+                            },
+                            {
+                                "label": "On Address",
+                                "path": "witness.outputs[0].recipient"
+                            }
```
Comment (ckorchane-ledger) [inline arbitrum/uniswap/eip712.json:108]: These fields need to be shown to the user too ?
```
                            {
                                "label": "Spender",
                                "path": "spender"
                            },
                            {
                                "format": "datetime",
                                "label": "Deadline",
                                "path": "deadline"
                            }
```

File: arbitrum/uniswap/eip712.json [hunk 2]
```diff
@@ -80,6 +80,451 @@
...
+                                "assetPath": "witness.inputToken",
+                                "format": "amount",
+                                "label": "To swap",
+                                "path": "witness.inputAmount"
+                            },
+                            {
+                                "assetPath": "witness.outputs[0].token",
+                                "format": "amount",
+                                "label": "Receive at Minimum",
+                                "path": "witness.outputs[0].amount"
```
Comment (ckorchane-ledger) [inline arbitrum/uniswap/eip712.json:103]: Do we show only the first output ? what if there are more than one ?

File: arbitrum/uniswap/eip712.json (hunk 1 above)
Comment (jnicoulaud-ledger) [reply arbitrum/uniswap/eip712.json:117]: change added

File: arbitrum/uniswap/eip712.json (hunk 2 above)
Comment (jnicoulaud-ledger) [reply arbitrum/uniswap/eip712.json]: we don't support it yet. we have decided to show the last instead

**Review by ckorchane-ledger — APPROVED** [review]

**Review by lcastillo-ledger — APPROVED** [review]

**Review by fsamier — APPROVED** [review]

**Pull request merged**